test/test_simulation_results.jl

```diff
@@ -1044,6 +1045,24 @@ end
     first_result = first(thermal_results)
     last_result = last(thermal_results)
 
+    println(list_aux_variable_keys(results_ed))
+
+    loss_factors = read_aux_variable(results_ed, PowerFlowLossFactors, ACBus)
+
+    # here we check if the loss factors are stored in the results, the values are tested in PowerFlows.jl
+    @test loss_factors !== nothing
+
+    loss_factors = first(
+        values(
+            PSI.read_results_with_keys(results_ed,
+                [PSI.AuxVarKey(PowerFlowLossFactors, ACBus)]),
+        ),
+    )
+
+    # here we check if the loss factors are stored in the results, the values are tested in PowerFlows.jl
+    @test loss_factors !== nothing
+    @test nrow(loss_factors) == 576
```

Comment:
Where does `576` come from? Can we do `... == length(get_components(...` instead to avoid hard-coding this?

Reply:
It is 48 * 12 (see also line 1067). Do you have an idea where it is best to read it from?

Reply:
`48 * 12` works for me